Source/ui_unix/openglwindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include "openglwindow.h"
+#include <QResizeEvent>
+
+OpenGLWindow::OpenGLWindow(QWindow *parent)
+    : QWindow(parent)
+{
+    QSurfaceFormat format;
+    format.setVersion(3, 2);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
@jpd002 I'll append the commit in few minutes, just as my laptop reboots into ubuntu.